gstd/gstd_symbolic_link.sh

```diff
@@ -4,4 +4,4 @@
 # $1: target
 # $2: link name
 rm -f ${DESTDIR}${MESON_INSTALL_PREFIX}/$2
-ln -s ${DESTDIR}${MESON_INSTALL_PREFIX}/$1 ${DESTDIR}${MESON_INSTALL_PREFIX}/$2
+ln -s --relative ${DESTDIR}${MESON_INSTALL_PREFIX}/$1 ${DESTDIR}${MESON_INSTALL_PREFIX}/$2
```

Comment:
Please check my comment about `--relative` and portability.

Reply:
reverted to original